steampy/client.py

```diff
@@ -278,7 +278,7 @@ def make_offer_with_url(self, items_from_me: List[Asset], items_from_them: List[
         }
         headers = {'Referer': self.COMMUNITY_URL + urlparse.urlparse(trade_offer_url).path,
                    'Origin': self.COMMUNITY_URL}
-        response = self._session.post(url, data=params, headers=headers).json()
+        response = self._session.post(url, data=params, headers=headers).json() or {}
```

Comment:
When the sessionid cookies is missing the status code is 401 and the text is null.
Because of that the variable "response" is null and the line below raises an exception (the getter).
I think my fix is the expected behavior.